modules/ipi-install-bmc-addressing-for-dell-idrac.adoc

```diff
@@ -124,7 +124,7 @@ platform:
 
 [NOTE]
 ====
-Currently, Redfish is only supported on Dell hardware with iDRAC firmware versions `4.20.20.20` through `04.40.00.00` for installer-provisioned installations on bare metal deployments. There is a known issue with version `04.40.00.00`. With iDRAC 9 firmware version `04.40.00.00`, the Virtual Console plug-in defaults to `eHTML5`, which causes problems with the *InsertVirtualMedia* workflow. Set the plug-in to `HTML5` to avoid this issue. The menu path is: *Configuration* -> *Virtual console* -> *Plug-in Type* -> *HTML5* .
+There is a known issue on Dell iDRAC 9 with firmware version `04.40.00.00` or later for installer-provisioned installations on bare metal deployments. The Virtual Console plug-in defaults to `eHTML5`, which causes problems with the *InsertVirtualMedia* workflow. Set the plug-in to `HTML5` to avoid this issue. The menu path is: *Configuration* -> *Virtual console* -> *Plug-in Type* -> *HTML5* .
 
 Ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: *Configuration* -> *Virtual Media* -> *Attach Mode* -> *AutoAttach* .
```

Comment:
This might be out of context for this PR, but I'll add a comment anyway for reference. You could update "AutoAttach" in the first line to use the same format as the menu path reference to that in the subsequent line.

Also, does "AutoAttach" refer to a UI object or a code reference? If it is a UI object reference you could use bold font instead of using back-ticks for both instances of that string.